app/views/shared/_flash.html.haml

```diff
@@ -1,5 +1,5 @@
 - if flash[:notice]
-  .notification= flash[:notice]
+  .notification= flash[:notice].html_safe
```

Comment:
What is this change for?

Reply:
oh I cherry-picked this from the [previous PR](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/4914/files#diff-62220853e559babd01ed5e0d5717a56f1ab4f0a8d1f0d8907e48dbed27266501). 
We are using `flash[:notice]` for rendering the success message in the controller, and that message has a link to the new course (which is added using `<a>` tag). It looks like we need to add the `html_safe` to indicate that the string should be considered safe HTML content and should not be escaped when rendered in a view. Without the `html_safe`, the link doesn't render properly.